### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by Kossecki**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -154,28 +154,28 @@ <h4 class="project-title">Super Awesome Work</h4>
...
-                        <div class="column column-centered column-quarter"><img src="voigtlander.svg" alt="" /></div>
+                        <div class="column column-centered column-quarter"><img src="com.svg" alt="Circus of Magazines logo" /></div>
+                        <div class="column column-centered column-quarter"><img src="voigtlander.svg" alt="Voigtlander logo" /></div>
                         <div class="column column-centered column-quarter">
-                            <img src="hok.svg" class="clients-logo-smaller" alt="" />
+                            <img src="hok.svg" class="clients-logo-smaller" alt="HOK Hellmut Obota&amp;Kassabaum logo" />
                         </div>
                         <div class="column column-centered column-quarter">
-                            <img src="sumop.svg" class="clients-logo-smaller" alt="" />
+                            <img src="sumop.svg" class="clients-logo-smaller" alt="Sumo Power logo" />
```
Comment (benabraham) [inline index.html]: “logo” might be unnecessary in alt. It's true but also images say: “our client is Voigtlander“ so “Voigtlander” should be enough.

File: index.html [hunk 2]
```diff
@@ -128,19 +128,19 @@ <h4 class="project-title">Super Awesome Work</h4>
...
-                                <img src="https://source.unsplash.com/Alm7tmPVmSU/500x450" alt="" class="project-image" />
+                                <img src="https://source.unsplash.com/Alm7tmPVmSU/500x450" alt="creative agency office" class="project-image" />
                             </div>
                             <div class="project-four">
-                                <img src="https://source.unsplash.com/IHtXWlztoMU/420x450" alt="" class="project-image" />
+                                <img src="https://source.unsplash.com/aS4Duj2j7r4/420x450" alt="camera studio set up" class="project-image" />
                             </div>
                             <div class="project-five">
-                                <img src="https://source.unsplash.com/5QgIuuBxKwM/500x450" alt="" class="project-image" />
+                                <img src="https://source.unsplash.com/5QgIuuBxKwM/500x450" alt="design meeting" class="project-image" />
```
Comment (benabraham) [inline index.html]: resolutions should be doubled to work on high DPI displays

File: styles.css [hunk 3]
```diff
@@ -10,8 +10,31 @@ html {
...
 
+p,
+a {
+    margin-top: 0;
+    margin-bottom: 0;
+    color: #181a1f;
+    font-size: 1.25rem;
+    font-weight: 400;
+    line-height: 1.25;
+    letter-spacing: 0.0752em;
```
Comment (benabraham) [inline styles.css]: 0.075em?

File: styles.css [hunk 4]
```diff
@@ -10,8 +10,31 @@ html {
...
+}
+
+h4 {
+    margin-top: 0;
+    margin-bottom: 1.7rem;
+    color: #ffdb00;
+    font-size: 3.125rem;
+    line-height: 1.53;
+    text-transform: uppercase;
+    letter-spacing: 0.07488em;
```
Comment (benabraham) [inline styles.css]: 0.075em?

File: styles.css [hunk 5]
```diff
@@ -44,7 +67,7 @@ button {
     padding: 1.5rem 2rem;
     text-align: center;
     font-size: 1.063rem;
-    letter-spacing: 0.079rem;
+    letter-spacing: 0.0743em;
```
Comment (benabraham) [inline styles.css]: 0.075em?

File: styles.css [hunk 6]
```diff
@@ -365,10 +373,11 @@ h4 {
     color: #181a1f;
     background-color: #ffda00;
     font-size: 1.875rem;
-    letter-spacing: 0.141rem;
+    letter-spacing: 0.0752em;
```
Comment (benabraham) [inline styles.css]: 0.075em?

**Comment by benabraham** [comment]
Also please use classes in all cases for input and textarea (lookup _input_ and _textarea_ in CSS…)

File: index.html (hunk 1 above)
Comment (Kossecki) [reply index.html]: Good point.

File: styles.css (hunk 3 above)
Comment (Kossecki) [reply styles.css]: Duh, 0.075em is the magic number.

File: index.html (hunk 2 above)
Comment (benabraham) [reply index.html]: What about this? 500x450 > 1000x900 etc.

**Comment by benabraham** [comment]
Also: javascript emulating project loading stopped working.

**Pull request merged**